services/privacy-analytics-service/src/privacy/DifferentialPrivacy.ts

```
@@ -0,0 +1,557 @@
+        start.setMinutes(0, 0, 0);
+        end.setMinutes(0, 0, 0);
+        end.setHours(end.getHours() + 1);
+        break;
+      case 'day':
+        start.setHours(0, 0, 0, 0);
+        end.setHours(0, 0, 0, 0);
+        end.setDate(end.getDate() + 1);
+        break;
+      case 'week':
+        const dayOfWeek = start.getDay();
+        start.setDate(start.getDate() - dayOfWeek);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)